modules/dnn/src/layers/slice_layer.cpp

```diff
@@ -114,7 +114,8 @@ class SliceLayerImpl : public SliceLayer
     virtual bool supportBackend(int backendId) CV_OVERRIDE
     {
         return backendId == DNN_BACKEND_OPENCV ||
-               ((backendId == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019 || backendId == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH) &&
+               (backendId == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH && sliceRanges.size() == 1) ||
```

Comment:
Did we have a test with `sliceRanges.size() > 1` before?

Reply:
Yes, Test_Caffe_layers.Reshape_Split_Slice: sliceRanges.size() = 3

Reply:
But why we need this condition in this PR?

Reply:
We enable case with ` sliceRanges.size() =1` and with any` sliceRanges[0].size()`  (not just `sliceRanges[0].size() == 4`)

Reply:
I mean this PR introduces limited usage of Slice layer with nGraph because of new mode of Mean layer?

Reply:
Oh I got it (just need to open an entire file to see). Thanks!